src/lib/db/event-store.ts

```diff
@@ -115,50 +116,44 @@ class EventStore extends EventEmitter implements IEventStore {
         }
     }
 
-    async getEventsFilterByType(name: string): Promise<IEvent[]> {
```

Comment:
This `getEventsFilterByType` method was only used in tests. 

The removal of its `DROP_FEATURES` logic does not seem to have any effect.